db/migrate/20200430132539_remove_assignments_progress_cache_and_es_deadline_field.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveAssignmentsProgressCacheAndEsDeadlineField < ActiveRecord::Migration
+  def change
+  	# calling reindex to remove `deadline` field
+    CheckElasticSearchModel.reindex_es_data
```

Comment:
How long will it take? If too long, please move to a rake task under `lib/tasks/migrate`.

Reply:
Can't we write an ES query to remove the `deadline` field?

Reply:
no it'll not take long time and it's a zero downtime 
ES query will remove deadline data and keep the field mapping to remove the mapping we need to re-index es data